gpflow/conditionals.py

```diff
@@ -74,6 +74,25 @@ def _conditional(Xnew, feat, kern, f, *, full_cov=False, full_output_cov=False,
     return fmean, _expand_independent_outputs(fvar, full_cov, full_output_cov)
 
 
+# def broadcast_rowwise_over_leading_dims(fn, M, output=-1):
```

Comment:
don't add commented-out code ...

Reply:
It thought I'd got rid of this already but I'll do so again if not

Reply:
Require change